helm/rook-operator/charts/monitoring/values.yaml

```diff
@@ -0,0 +1,7 @@
+# prepend all rule names for easier identification
+nameStub: "rook-operator"
+
+rules:
+  # rules must currently be deployed to the same namespace
+  # as prometheus-operator.
+  deploymentNamespace: "monitoring"
```

Comment:
this restriction can be removed when prometheus-operator is allowed to watch other namespaces for rule definitions.